types/eslint-plugin-jsx-a11y/index.d.ts

```diff
@@ -6,8 +6,8 @@ declare const eslintPluginJsxA11y: ESLint.Plugin & {
         strict: Linter.LegacyConfig;
     };
     flatConfigs: {
-        recommended: Linter.FlatConfig;
-        strict: Linter.FlatConfig;
+        recommended: Linter.Config;
+        strict: Linter.Config;
```

Comment:
these are flat configs, why should the type not be "FlatConfig"?

Reply:
ESLint changed their FlatConfig, they want to use Config as default name as mentioned at this link:
https://github.com/eslint/eslint/blob/d46059410a0e02b98067aa31975c25fd8d0d1c2b/lib/types/index.d.ts#L1319

Reply:
then what's the type for an eslintrc config?

Reply:
From this [link](https://eslint.org/docs/latest/use/configure/configuration-files#typescript-configuration-files) it's the `Linter.Config`
<img width="699" alt="image" src="https://github.com/user-attachments/assets/a7013cb1-81f5-413b-82bb-0086a453c01c">
To illustrate this better I am sending a screenshot on my VSCode which is using both Config and FlatConfig
<img width="412" alt="image" src="https://github.com/user-attachments/assets/f38df292-b398-43cd-98dc-d2def2014471">

Reply:
i guess i'm confused, a flat config and an eslintrc config can't possibly have the same type since they have different schemas.

Reply:
Now, with the new flat config, the eslint config file is just `export default` an array of flatConfig objects. The flatConfig is the default now, so they don't want to call it "flat" anymore, just the "config." The old config is called "legacy."

Reply:
ok, so eslintrc config (legacy, but that’s an editorial name so i don’t use it) is “LegacyConfig”?

Reply:
That's exactly, the v8 use `.eslintrc.*` which exports an object of type `LegacyConfig`
v9 use `eslint.config.*` which exports an array of `Config`